src/MainWindow.cpp

```diff
@@ -0,0 +1,936 @@
+/*##################################################################################
+#   File name:          MainWindow.cpp
+#   Created on:         08/30/2024
+#   Author:             stwomey
+#   Description:        Header file for the MainWindow
+#   Change History:     See https://github.com/SeanTwomey9/PersonalFinanceTool
+#   Relative Location:  PersonalFinanceTool/src/
+##################################################################################*/
+
+#include "MainWindow.h"
+#include "BillWidget.h"
+
+#include <QFile>
+#include <QDir>
+#include <QMessageBox>
+#include <QInputDialog>
+#include <QApplication>
+#include <QComboBox>
+
+MainWindow::MainWindow()
+{
+    // Connection to terminate application when certain conditions are met
+    connect(this, SIGNAL(conditionToTerminateMet()), this, SLOT(terminateApplication()), Qt::QueuedConnection);
+
+    // Set the title of the MainWindow
+    this->setWindowTitle(m_APP_NAME);
+
+    // Size the MainWindow appropriately
+    this->setGeometry(0, 0, 500, 520);
+
+    // Create the BillWidget
+    m_billWidget = new BillWidget(this);
+
+    // When the BillWidget's Close button is clicked, terminate the application
```

Comment:
Same thing with the naming, either name these widgets as *BillWidget* or *bill widget* but don't mix and match.

Reply:
Used *BillWidget* throughout comments as this is an actual class object.